mailserver/mailserver.go

```diff
@@ -444,17 +438,29 @@ func (s *WMailServer) createIterator(lower, upper uint32, cursor []byte) iterato
 // processRequestInBundles processes envelopes using an iterator and passes them
 // to the output channel in bundles.
 func (s *WMailServer) processRequestInBundles(
-	iter iterator.Iterator, bloom []byte, limit int, output chan<- []*whisper.Envelope,
+	iter iterator.Iterator,
+	bloom []byte,
+	limit int,
+	output chan<- []*whisper.Envelope,
 ) ([]byte, common.Hash) {
 	var (
 		bundle                 []*whisper.Envelope
 		bundleSize             uint32
+		batches                [][]*whisper.Envelope
 		processedEnvelopes     int
 		processedEnvelopesSize int64
 		nextCursor             []byte
 		lastEnvelopeHash       common.Hash
 	)
 
+	log.Debug("[mailserver:processRequestInBundles] processing request", "limit", limit)
```

Comment:
to improve logging filtering, I would also add a `requestID` of a kind, for instance `requestID := fmt.Sprintf("%d-%d", time.Time.Unix(), rand.Intn(1000))` and add it to the logs